README.md

```diff
@@ -1,13 +1,13 @@
 # Formal ledger specifications
 
-This repository contains the formal ledger specifications that are intended to eventually replace the existing formal specifications of the Cardano ledger found [here](https://github.com/input-output-hk/cardano-ledger). This project is currently incomplete and work in progress.
+This repository contains the formal ledger specifications that are intended to eventually replace the existing formal specifications of the Cardano ledger found [here](https://github.com/intersectmbo/cardano-ledger). This project is currently incomplete and work in progress.
```

Comment:
Better use capitalised 'IntersectMBO' here to be consistent with the rest of the document and make grep-/sed-ing easier in the future.